### DISCUSSION TITLE
hello! it's my first time here, so... can someone explain this engine to me? i want to learn about it

### DISCUSSION THREAD

**Discussion by Benby3** [Leather128/LeathersFunkinEngine#173] [General]
yeah i really wants to learn about this engine

### sorry for my bad english, cuz i brazillian and dont know how to talk english soo much.

**Comment by foxygood** [comment]
hey i am brazilian too!

**Reply by Benby3** [reply]
SERIO? '-'

**Reply by foxygood** [reply]
sim

**Comment by ghost** [comment]
Sure, Leather Engine is like Psych Engine but with features like

- Extra Keys (1 to 18 Keys)
- Better systems for stage
- Prototype title screen (is in options - misc)
- Cutscenes Play on Story, Freeplay and both (is in options - misc)
- Play as BF or opponent (is in options - misc)
- UI Skin (is in options - Graphics - Note Options)
- The opponent has a splash note now (is in options - Graphics - Note Options)
- Tools to make character creator, animated debug and charter
- In the pause menu there restart song with cutscenes and options
- in options there is 
- Bot
- Auto Restart
- No Miss
- Ghost Tapping
- No Death
- There is a new (idk what call) in game called Marvelous when you hit perfect
- there Replays
- you can full screen with F11

there more in the engine and is pretty amazing

and there is a build in [actions](https://github.com/Leather128/LeatherEngine/actions), they are not release but you can play test.

also if you want to make a mod go to this [link](https://github.com/Leather128/LeatherEngine/wiki)

( also i'm speak Spanish but I can speak english.)

**Reply by Benby3** [reply]
THANKS A LOT!
this help me soo much!
now i love this engine, and i love all peoples are here.
thanks soo much to explain more to me! =D

**Reply by ghost** [reply]
you welcome :D

**Comment by zephyr-zeph** [comment]
yo its you, from psych engine wiki!

**Reply by Benby3** [reply]
yeah it's me
